typescript/sdk/src/ism/HyperlaneIsmFactory.ts

```diff
@@ -122,6 +124,24 @@ export class HyperlaneIsmFactory extends HyperlaneApp<IsmFactoryFactories> {
     return IMultisigIsm__factory.connect(address, signer);
   }
 
+  public async deployMerkleRootMultisigIsm(
```

Comment:
why not just use `deployMultisigIsm`?

Reply:
doesn't return StaticMerkleRootMultisigIsm

Reply:
why do you need that? in any case this is a big dupe of the function right above it